pkg/healthautoexport/types.go

```diff
@@ -140,8 +167,34 @@ func (m *Metric) UnmarshalJSON(bytes []byte) error {
 			return err
 		}
 		m.Datapoints = d
-		return nil
 	}
+
+	return nil
+}
+
+func (m *Metric) unmarshalSleepAnalysis(data []byte) bool {
+	// Try to unmarshal as SleepAnalysis first.
+	// However, it seems that later versions of Apple Health no longer expose this information.
```

Comment:
Can you explain this comment? I'm still getting non-aggregated sleep metrics with no issues with v0.3.0.

Reply:
This comment is no longer relevant, previously I mixed up the data I received and wrongly assumed that non-aggregated data was no longer exported.